src/søknad/steg/2-bosituasjon/SøkerSkalFlytteSammenEllerFåSamboer.tsx

```diff
@@ -111,21 +88,27 @@ const SøkerSkalFlytteSammenEllerFåSamboer: FC<{ intl: IntlShape }> = ({
               }
               tekstid={'datovelger.nårSkalDetteSkje'}
               datobegrensning={DatoBegrensning.FremtidigeDatoer}
-              settDato={(e) =>
-                settDatoSøkerSkalGifteSegEllerBliSamboer(e, datovelgerTekst)
-              }
-            />
-          </KomponentGruppe>
-          <KomponentGruppe>
-            <OmSamboerenDin
-              tittel={'bosituasjon.tittel.hvemSkalSøkerGifteEllerBliSamboerMed'}
-              ekteskapsLiknendeForhold={false}
+              settDato={(e) => {
+                settDatoSøkerSkalGifteSegEllerBliSamboer(e, datovelgerTekst);
+              }}
             />
           </KomponentGruppe>
+          {datoSkalGifteSegEllerBliSamboer && (
+            <KomponentGruppe>
+              <OmSamboerenDin
```

Comment:
Hvorfor må man legge denne i en komponentgruppe? :)

Reply:
nevermind, det er kanskje for konsistent struktur mtp styling?

Reply:
Yup :) 
SeksjonsGruppe, KomponentGruppe og FeltGruppe er egentlig bare styling wrappere for å gi konsistent struktur & styling. Tuva er litt usikker på spacing mellom spørsmålene og sånt ennå, så hvis vi bruker disse blir det litt lettere å gå i gjennom og fikse senere. 
Gir også litt mer forståelig html-semantikk :) 
De er ikke helt satt i stein siden designet ikke er helt satt is sten :)